Moda.Infrastructure/src/Moda.Infrastructure/Persistence/Configuration/LinksConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Moda.Links.Models;
+
+namespace Moda.Infrastructure.Persistence.Configuration;
+public class LinkConfiguration : IEntityTypeConfiguration<Link>
+{
+    public void Configure(EntityTypeBuilder<Link> builder)
+    {
+        builder.ToTable(name: "Links", SchemaNames.Links);
+
+        builder.HasKey(o => o.Id);
+
+        builder.HasIndex(o => o.Id)
+            .IncludeProperties(o => new { o.ObjectId, o.Name, o.Url });
+        builder.HasIndex(o => o.ObjectId)
+            .IncludeProperties(o => new { o.Id, o.Name, o.Url});
+
+        builder.Property(o => o.ObjectId).IsRequired();
+        builder.Property(o => o.Name).IsRequired().HasMaxLength(128);
+        builder.Property(o => o.Url).IsRequired();
```

Comment:
Not really a big deal, but is nvarchar(max) needed for a url? I can't imagine a url would be more than 1024 characters.

Reply:
I originally thought the same thing.  After some research I found that there doesn't seem to be a max size and that most of the modern browsers support more than 64k characters.  Chrome supports over 2 million.  Since this data is only queryable by the objectId or specific Id I figured it would be fine without a limit.